docs/pages/desktop-access/rbac.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Desktop Access Access Controls
```

Comment:
```suggestion
title: Role-Based Access Control for Desktop Access
```
I thought the double-"Access" looked like a typo at first—I've suggested an alternative.